README_FA.md

```diff
@@ -0,0 +1,22 @@
+<div dir="rtl">
+
+# پیوستن به تیم ما
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
```

Comment:
Also this `ALL-CONTRIBUTORS` section needs to be removed since it is autogenerated and only appears on the English README

Reply:
Also I should remove its badge?

Reply:
Yes the 3 lines need to be removed